course_content/notebooks/iris_course/1.The_Iris_Cube.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": null,
```

Comment:
Do we want to remove the execution of each cell?
I had added it was then it meant you could look through the notebook without needing to run it.

Reply:
Just noticed, I never replied to this.
I actually think it is better to have the users 'discover' all the outputs.
It also makes all the management operations on GitHub far easier (i.e. seeing changes + reviewing).